Missionframework/modules/11_permission/fnc/fn_permission_checkPermission.sqf

```diff
@@ -1,36 +1,46 @@
+#include "script_component.hpp"
 /*
     KPLIB_fnc_permission_checkPermission
 
     File: fn_permission_checkPermission.sqf
     Author: KP Liberation Dev Team - https://github.com/KillahPotatoes
     Date: 2018-12-09
-    Last Update: 2018-12-28
+    Last Update: 2019-05-01
     License: GNU General Public License v3.0 - https://www.gnu.org/licenses/gpl-3.0.html
 
     Description:
         Checks the given permission and executes the registered code.
 
     Parameter(s):
-        _permission - Permission name [STRING, defaults to ""]
+        _permission - Permission name                                                           [STRING, defaults to ""]
+        _type       - Operator which will be used for the code check (false = ||, true = &&)    [BOOL, defaults to false]
+        _target     - Target player                                                             [OBJECT, defaults to objNull]
 
     Returns:
         Result of the registered code [BOOL]
 */
 
 params [
-    ["_permission", "", [""]]
+    ["_permission", "", [""]],
+    ["_type", false, [false]],
+    ["_target", player, [objNull]]
 ];
 
 if (!KPLIB_param_permission || serverCommandAvailable "#kick") exitWith {
     true
 };
 
-private _results = [];
-private _data = KPLIB_permission_data getVariable [toLower _permission, []];
+// Variables
+private _data = PGVAR(toLower _permission, []);
+private _codeResult = false;
+private _result = [_permission, _target] call KPLIB_fnc_permission_getPermission;
 
 // Execute the registered code
-{
-    _results pushBack ([] call _x);
-} forEach (_data select 0);
+_codeResult = [] call (_data select 0);
```

Comment:
Isn't only one of registered handler functions called after this change? Is this intentional?

Reply:
I've deleted the multiple code option for the permission handlers as it's not very useful.